### ISSUE TITLE
FVS: Update 'Add checked items to tree species table'

### ISSUE THREAD

**Issue by lbross** [USFS-PNW/Fia-Biosum-Manager#157]
For FIA spcd that aren't in the fvs_tree_species table, this function will fail because the fvs_input_spcd column is null in the audit grid. For FIA spcd < 300 that are missing from the fvs_tree_species table set the fvs_input_spcd to 298. For all other FIA spcd that are missing from the fvs_tree_species table set the fvs_input_spcd to 998.

**Comment by lbross** [comment]
There was a bug in the tree_species audit when the audit is launched from the FVS button grouping. This audit returns 2, not 4, fields unlike the Processor button. When the analyst tried to edit the newly added row, an exception was thrown because one of the fields was unexpectedly null. Until this fix is implemented in an .msi, analysts should use the Tree Species audit from the Processor button grouping.
I also added code to set the default fvs_input_spcd as directed in the original request. This happens on the tree species edit form.
For documentation sake, when a tree species is added from the FVS button grouping, the new species needs to be edited to complete some fields that aren't returned in that audit: fvs_input_spcd.
This code has been merged to the master and issue_130 branches and is ready for deployment.
